src/pages/ContentfulPage.vue

```diff
@@ -292,6 +294,7 @@ export default {
 				this.pageError = true;
 			} else {
 				this.title = (pageData?.page?.pageLayout?.pageTitle || pageData?.page?.pageTitle) ?? undefined;
+				this.pageBackgroundColor = pageData?.page?.pageLayout?.pageBackgroundColor ?? undefined;
```

Comment:
The default value here - `undefined`, should probably match the component default in `data:` above - ''. I think either undefined or '' would work